modules/getting-started/pages/choose-your-next-steps.adoc

```diff
@@ -44,7 +44,7 @@ For information on doing so, see the section xref:n1ql:n1ql-rest-api/index.adoc[
 The Couchbase SDK exists in different versions: for node.js, Java, .NET, Go, C, PHP, and Python.
 
 Under the *DEVELOPERS* tab in the vertical navigation-bar to the left, you'll find detailed information on all aspects of the Couchbase SDK, and how to use it.
-First, take a look at the section xref:2.4@nodejs-sdk::start-using-sdk.adoc[Start Using the SDK].
+First, take a look at the section xref:java-sdk::start-using-sdk.adoc[Start Using the SDK].
 Towards the upper right of that page is a pull-down menu, which allows you to select a language.
```

Comment:
_Towards the upper right of that page is a pull-down menu, which allows you to select a language._
Should be
_At the top of that page is a pull-down menu, which allows you to select a language._
For the new docset.

Reply:
Done